examples/nextjs/src/components/my-page.js

```diff
@@ -14,46 +14,43 @@ import { DotcmsLayout } from "@dotcms/react";
 
 import { DotExperimentsProvider } from "@dotcms/experiments";
 
-
 export function MyPage({ data, nav }) {
-  const { refresh, replace} = useRouter();
+  const { refresh, replace } = useRouter();
   const pathname = usePathname();
 
   const experimentConfig = {
-    apiKey: process.env.NEXT_PUBLIC_EXPERIMENTS_API_KEY ?? '' , //We dont want to run experiments all the time
-    server: process.env.NEXT_PUBLIC_DOTCMS_HOST ,
+    apiKey: process.env.NEXT_PUBLIC_EXPERIMENTS_API_KEY ?? "", //We dont want to run experiments all the time
```

Comment:
I think we should add a prettier file to this project. We all have different formatting in our IDEs

Reply:
What do you think? @oidacra @fmontes

Reply:
Yes. Could be a good idea.